scalafmt-tests/src/test/resources/rewrite/RedundantParens.stat

```diff
@@ -271,3 +271,15 @@ object a {
 object a {
   print("") shouldBe (())
 }
+<<< yield without a space
+object a {
+  val x = for {
+    a <- b
+  } yield(a)
```

Comment:
is this specific to `yield`? what about:

```
  a match {
    case a if(a) => a
  }
```

Reply:
I'm testing it as we speak, I'll open another MR if it also fails there

Reply:
Ahh I just saw you PR, great. I was not sure if extra spaces would be removed later on

Reply:
i already fixed it, see linked pr.

Reply:
and thank you for not only reporting a problem but also fixing it. doesn't happen too often :)